grafana/resource_data_source_test.go

```diff
@@ -248,6 +248,9 @@ resource "grafana_data_source" "testdata" {
 			json_data {
 				http_method = "GET"
 				query_timeout = "1"
+                sigv4_auth   = true
+	            sigv4_auth_type = "default"
+	            sigv4_region    = "eu-west-1"
```

Comment:
Whitespace is a bit wonky here.
```suggestion
				sigv4_auth   = true
				sigv4_auth_type = "default"
				sigv4_region    = "eu-west-1"

```